README.md

```diff
@@ -8,6 +8,21 @@ Installs and configures the Cockpit Web Console for distributions that support i
   - RHEL/CentOS 7.x depend on the Extras repository being enabled.
   - Recommended to use [`linux-system-roles.firewall`](https://github.com/linux-system-roles/firewall/) to make the Web Console available remotely.
 
+  - The role requires the `firewall` role and the `selinux` role from the
+    `fedora.linux_system_roles` collection, if `cockpit_manage_firewall`
+    and `cockpit_manage_selinux` is set to true, respectively.
+    (Please see also `cockpit_manage_firewall` and `cockpit_manage_selinux`
+     in [`Role Variables`](#role-variables).
+
+    If the `cockpit` is a role from the `fedora.linux_system_roles`
+    collection or from the Fedora RPM package, the requirement is already
+    satisfied.
+
+    Otherwise, please run the following command line to install the collection.
+    ```
+    ansible-galaxy collection install -r meta/collection-requirements.yml
```

Comment:
> This would only work if you are in a git checkout of this repo, right?

It is also present in the meta directory of the role if you install from Galaxy.

> Perhaps this can be spelled out instead of the "Otherwise"?

I'm assuming it is implied from the context, but if you can suggest something that isn't too verbose, we can change this and all of the other roles where we already have this exact same verbiage.

Reply:
If it is implied for users who are more familiar with this, then fine of course. I was just bringing it up as it sounds confusing to me -- this relative file path *definitively* makes some assumption of what directory you are in. That is fine for a git clone, but this README also ends up in the official documentation, I suppose.

Reply:
> If it is implied for users who are more familiar with this, then fine of course. I was just bringing it up as it sounds confusing to me -- this relative file path _definitively_ makes some assumption of what directory you are in.

It might also imply "the `meta` directory of the role source from wherever you are using the role".  Again, if you can suggest something more appropriate, we will consider it.

> That is fine for a git clone, but this README also ends up in the official documentation, I suppose.

Depends on what you mean by "official documentation".  The collection install is *only* needed if you are using the role from Galaxy or github.  If you are using the role from any other context - from the collection as fedora.linux_system_roles.cockpit - from any of the RPM distributions - from Automation Hub - you *do not* need to do this step.

Reply:
Ack, thanks @richm for the clarification! Makes sense then.